docs/user/install.rst

```diff
@@ -133,6 +133,18 @@ For a more in-depth overview of available servers, see also:
 
     $ pip install [uvicorn|daphne|hypercorn]
 
+.. note::
+
+    By default, the ``uvicorn`` package comes only with a minimal set of
+    pure-Python dependencies.
+    For CPython-based production deployments, you can install Uvicorn along
+    with faster alternatives such as ``uvloop`` and ``httptools``::
```

Comment:
Faster alternative of what? I think we should specify that we are talking about the asyncio loop here

Reply:
Thanks, I'll try rewording (unless you have a good suggestion).